youtube_transcript_api/_api.py

```diff
@@ -100,12 +101,19 @@ class _TranscriptFetcher():
     API_BASE_URL = 'https://www.youtube.com/api/{api_url}'
     LANGUAGE_REGEX = re.compile(r'(&lang=.*&)|(&lang=.*)')
 
-    def __init__(self, video_id, languages):
+    def __init__(self, video_id, languages, proxy):
         self.video_id = video_id
         self.languages = languages
+        if isinstance(proxy, str):
```

Comment:
I'd prefer the input of this to be consistently typed. The handling of different inputs should be done while parsing the arguments.